### PR TITLE
Create utility class for compareXml/compareTxt for refactoring purposes

### LINKED ISSUE #2208: Create utility class for compareXml/compareTxt
* **Do you want to request a *feature* or report a *bug*?**
Refactoring

* **What is the current behavior?**
Utility classes `compareXml` and `compareTxt` are in `AbstractConverterTest`

* **What is the expected behavior?**
These methods should be in a utility class `ComparisonUtils`

* **What is the motivation / use case for changing the behavior?**
Cleaner design

* **Please tell us about your environment:**
  - PowSyBl Version: 4.10.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by sfid98**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
Fix #2208 

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Refactoring

### CODE REVIEW

File: commons/src/test/java/com/powsybl/commons/ComparisonUtils.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
+package com.powsybl.commons;
```
Comment (geofjamg) [inline commons/src/test/java/com/powsybl/commons/ComparisonUtils.java:1]: Don't forget copyright and authorship as other files.

**Review by miovd — COMMENTED** [review]
Thank you for your work, the development is great for me! As @geofjamg, only copyright and authorship are missing. After that, it will be ready to be merged!

File: commons/src/test/java/com/powsybl/commons/ComparisonUtils.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+import javax.xml.transform.Source;
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+
+public class ComparisonUtils {
```
Comment (miovd) [inline commons/src/test/java/com/powsybl/commons/ComparisonUtils.java:17]: This class should be final with a private constructor (see checkstyle issues for more details)

**Review by miovd — APPROVED** [review]
Okay for me once the conflict is resolved. Thanks!

**Review by miovd — COMMENTED** [review]
Just a last thing! It seems that `AbstractConverterTest.compareXml` is still used in `EncodingTest` (it was merged after this PR was made). Could you please just fix that?

**Review by miovd — APPROVED** [review]

**Comment by annetill** [comment]
@sfid98 can you please resolve the conflict as we want to publish a release candidate on Wednesday. Thanks a lot!

**Review by annetill — APPROVED** [review]

**Comment by miovd** [comment]
@sfid98 Hi! Sorry about the merging delay! Is it possible for you to resolve the conflict? I'm merging this PR as soon as it is done.

**Pull request merged**